docs/swagger.yaml

```diff
@@ -286,7 +391,7 @@ components:
                   quantity: '2.5'
                   quantityUnit: KG
                 discount: 0
-                productUrl: https://www.vipps.no/store/socks
+                productUrl: 'https://www.vipps.no/store/socks'
```

Comment:
Compared to line 337 - a bit inconsistent with '' and "

Reply:
Skjønner ikke denne

Reply:
Fnuttene som er rundt teksten
Er kanskje ikke veldig viktig da
<img width="860" alt="image" src="https://user-images.githubusercontent.com/47554984/160612665-f1791be2-c78d-4c8f-9398-aaa247af8131.png">

Reply:
Ok, nå ser jeg. Jeg har ikke gjort noe. Det er stoplight studio som endrer på dette